components/UpdateProfileForm/steps/MilitaryDetails.js

```diff
@@ -8,99 +8,95 @@ import Input from 'components/Form/Input/Input';
 import Select from 'components/Form/Select/Select';
 import styles from './_steps.css';
 
-class MilitaryDetails extends React.Component {
-  static propTypes = {
-    isSubmitting: bool,
-  };
+MilitaryDetails.propTypes = {
+  isSubmitting: bool,
+};
 
-  static defaultProps = {
-    isSubmitting: false,
-  };
+MilitaryDetails.defaultProps = {
+  isSubmitting: false,
+};
+MilitaryDetails.title = 'Military Details';
 
-  static title = 'Military Details';
+MilitaryDetails.validationSchema = Yup.object().shape({
+  // TODO: use when() on these
+  branchOfService: Yup.string()
+    .nullable()
+    .required(validationErrorMessages.required),
+  yearsOfService: Yup.number()
+    .nullable()
+    .positive('Enter a number between 1 and 40.')
+    .lessThan(41, 'Enter a number between 1 and 40.')
+    .required(validationErrorMessages.required),
+  payGrade: Yup.string()
+    .nullable()
+    .required(validationErrorMessages.required),
+});
 
-  static validationSchema = Yup.object().shape({
-    // TODO: use when() on these
-    branchOfService: Yup.string()
-      .nullable()
-      .required(validationErrorMessages.required),
-    yearsOfService: Yup.number()
-      .nullable()
-      .positive('Enter a number between 1 and 40.')
-      .lessThan(41, 'Enter a number between 1 and 40.')
-      .required(validationErrorMessages.required),
-    payGrade: Yup.string()
-      .nullable()
-      .required(validationErrorMessages.required),
-  });
+MilitaryDetails.initialValues = {
+  branchOfService: '',
+  yearsOfService: '',
+  payGrade: '',
+};
 
-  static initialValues = {
-    branchOfService: '',
-    yearsOfService: '',
-    payGrade: '',
-  };
+MilitaryDetails.submitHandler = async values => {
+  await updateUser(values);
+};
 
-  static submitHandler = async values => {
-    await updateUser(values);
-  };
+function MilitaryDetails(props) {
```

Comment:
Can you destructure here instead of making line 46?

Reply:
done